<launch>
    <!-- <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="log"></node> -->
    <node name="turtlesim" pkg="turtlesim" type="turtlesim_node" output="log"></node>
    <node name="mission_control" pkg="mission_control" type="mission_control_node" output="screen" >
        <rosparam command="load" file="$(find mission_control)/config/demo.yaml"/>
    </node>
    <node name="keyboard_control" pkg="mission_control" type="keyboard_control_node" output="log" ></node>
    <node name="waypoint" pkg="mission_control" type="waypoint_node" output="log" ></node>
</launch>